### PR TITLE
Upgrade to v8

### PR DESCRIPTION
**Pull request by janovesk**

### CODE REVIEW

File: src/TimeoutMigrationTool.Raven3.AcceptanceTests/TimeoutMigrationTool.Raven3.AcceptanceTests.csproj [hunk 1]
```diff
@@ -4,6 +4,8 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
 
     <IsPackable>false</IsPackable>
+
+    <RootNamespace>TimeoutMigrationTool.Raven3.AcceptanceTests</RootNamespace>
```
Comment (timbussmann) [inline src/TimeoutMigrationTool.Raven3.AcceptanceTests/TimeoutMigrationTool.Raven3.AcceptanceTests.csproj:8]: does that have to be specified? Given the project name, I'd expect that to be the default already?

File: src/TimeoutMigrationTool.Raven4.Core8.AcceptanceTests/ConfigureExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,24 @@
+﻿namespace TimeoutMigrationTool.Raven4.Core8.AcceptanceTests
```
Comment (timbussmann) [inline src/TimeoutMigrationTool.Raven4.Core8.AcceptanceTests/ConfigureExtensions.cs:1]: does this intentionally use a different namespace?

File: src/TimeoutMigrationTool.Raven3.AcceptanceTests/TimeoutMigrationTool.Raven3.AcceptanceTests.csproj (hunk 1 above)
Comment (janovesk) [reply src/TimeoutMigrationTool.Raven3.AcceptanceTests/TimeoutMigrationTool.Raven3.AcceptanceTests.csproj]: Fixed

File: src/TimeoutMigrationTool.Raven4.Core8.AcceptanceTests/ConfigureExtensions.cs (hunk 2 above)
Comment (janovesk) [reply src/TimeoutMigrationTool.Raven4.Core8.AcceptanceTests/ConfigureExtensions.cs:1]: It's not :)

**Review by timbussmann — APPROVED** [review]

**Pull request merged**